starboard/common/spin_lock.h

```diff
@@ -21,15 +21,15 @@
 #ifndef STARBOARD_COMMON_SPIN_LOCK_H_
 #define STARBOARD_COMMON_SPIN_LOCK_H_
 
-#include "starboard/atomic.h"
+#include <atomic>
 
 namespace starboard {
 
-const SbAtomic32 kSpinLockStateReleased = 0;
-const SbAtomic32 kSpinLockStateAcquired = 1;
+constexpr int kSpinLockStateReleased{0};
+constexpr int kSpinLockStateAcquired{1};
 
-void SpinLockAcquire(SbAtomic32* atomic);
-void SpinLockRelease(SbAtomic32* atomic);
+void SpinLockAcquire(std::atomic_int* atomic);
```

Comment:
atomic_int32_t?

Reply:
atomic_int32_t is optional type and might be not available for some platforms. I used atomic<int32_t> instead.